docs/installing-solidity.rst

```diff
@@ -6,6 +6,28 @@
 Installing Solidity
 ###################
 
+Versioning
```

Comment:
I think this should go to the end of the "Installing Solidity" section. If I click on "Installing Solidity", the first thing I want to read is where to download the latest binary, and not read up on versioning first :-)